lte/gateway/python/magma/mobilityd/ip_allocator_dhcp.py

```diff
@@ -42,8 +42,14 @@
 DEFAULT_DHCP_REQUEST_RETRY_FREQUENCY = 10
 DEFAULT_DHCP_REQUEST_RETRY_DELAY = 1
 LEASE_RENEW_WAIT_MIN = 200
-
-DHCP_HELPER_CLI = "dhcp_helper_cli.py"
+DHCP_HELPER_CLI = ""
+
+# TODO: Change this back to just "dhcp_helper_cli.py" in PR 14635
+if os.environ.get('MAGMA_ROOT') and \
```

Comment:
Doesn't the current solution set up a symlink in the dev VM so we wouldn't need this workaround to make development of the script easier?

Reply:
Good catch, seems like this snuck in again... I removed this change.